Wordle.Tests/WordleUnitTest.cs

```diff
@@ -1,9 +1,138 @@
 using Xunit;
+using Wordle.Domain;
+using static Wordle.Domain.Game;
 
 namespace Wordle.Tests;
 
 public class WordleUnitTests
 {
+    public static string answer = "ADEPT";
+
     [Fact]
     public void PassingTest() => Assert.True(true);
+
+    [Fact]
+    public void EvaluateGuess_WhenNoLetterIsInWord_ShouldReturnAllNotInWord()
+    {
+        var guess = "GROWN";
+        var game = new Wordle.Domain.Game();

```

Comment:
could this be pulled into somekind of setup? or does a new game need to be created for each test?

Reply:
So I had previously attempted to refactor this but was unable to get it working because I was still using `var`. I'm glad you mentioned it though because I tried it again with different syntax and it worked this time! Thanks for calling it out.